pettingzoo/utils/env_logger.py

```diff
@@ -61,20 +61,6 @@ def warn_action_out_of_bound(
             f"[WARNING]: Received an action {action} that was outside action space {action_space}. Environment is {backup_policy}"
         )
 
-    @staticmethod
```

Comment:
Why exactly was this stuff removed? Looks kind of messy code and isn’t done in the other wrappers so can probably see it just want to make sure it’s on purpose that it got removed

Reply:
It is dead code that is never called. I think it was maybe intended to be used in the OrderEnfocingWrapper - and maybe was used in an older version of it, but it's not used anymore. The code is specific to that wrapper, if it's not used there, there's no other place it would be useful.